apps/service_providers/llm_service/runnables.py

```diff
@@ -50,7 +50,9 @@ def create_experiment_runnable(
     """Create an experiment runnable based on the experiment configuration."""
 
     if assistant := experiment.assistant:
-        history_manager = ExperimentHistoryManager.for_assistant(session=session, experiment=experiment)
+        history_manager = ExperimentHistoryManager.for_assistant(
```

Comment:
Need to update the function signature so that `trace_service` is not `None` by default

Reply:
It is None in tests

Reply:
It should never be None in prod code - I can update the tests so that it is always required

Reply:
updated

Reply:
Thanks, and sorry for being so pedantic. I like the philosophy of not having code written in a certain way to accomodate tests...as far as reasonably possible of course. By all means do hold me to this standard as well, should we choose to go with it